### PR TITLE
CNN Architecture with 5 convolutional layers, followed by ReLu and max pooling layers 

### PR DESCRIPTION
**Pull request by Bsewani**
CNN Architecture with 5 convolutional layers, followed by ReLu and max pooing layers

### CODE REVIEW

File: CNN_Architecture.ipynb [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        "128 input channels are converted to 256 output channels, followed by batch normalization, ReLU, and max pooling."
+      ],
+      "metadata": {
+        "id": "ZN1c3yKPAJ5W"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class CustomModel(nn.Module):\n",
```
Comment (mohananup) [inline CNN_Architecture.ipynb:88]: we need to import nn module from torch

File: CNN_Architecture.ipynb [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        "    class CustomConvModel(nn.Module):\n",
```
Comment (mohananup) [inline CNN_Architecture.ipynb]: Is the nested class necessary?

File: CNN_Architecture.ipynb [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        "     def __init__(self):\n",
+        "       super(CustomModel.CustomConvModel, self).__init__()\n",
+        "       self.conv_layer = nn.Sequential(\n",
+        "            nn.Conv2d(1, 16, kernel_size=3, padding=1),\n",
+        "            nn.BatchNorm2d(num_features=16),\n",
```
Comment (mohananup) [inline CNN_Architecture.ipynb:93]: does it matter the sequence of applying BatchNorm2d then I guess we need to apply on the output of Conv2d

File: CNN_Architecture.ipynb (hunk 1 above)
Comment (Bsewani) [reply CNN_Architecture.ipynb:88]: I have updated.

File: CNN_Architecture.ipynb (hunk 2 above)
Comment (Bsewani) [reply CNN_Architecture.ipynb]: Updated the code with single class

File: CNN_Architecture.ipynb (hunk 3 above)
Comment (Bsewani) [reply CNN_Architecture.ipynb:93]: We can discuss this comment but in general - I have applied batch normalization immediately after each convolutional layer and before the activation function (ReLU).
